src/main/java/com/Elessar/app/util/kafka/ConsumerCreator.java

```diff
@@ -0,0 +1,30 @@
+package com.Elessar.app.util.kafka;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.LongDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+
+import java.util.Collections;
+import java.util.Properties;
+
+/**
+ * Created by Hans on 3/17/19.
+ */
+public class ConsumerCreator {
+    public static Consumer<Long, String> create(String topic) {
```

Comment:
You should just make the return type generic instead of hard-coding <Long, String>, same for the producer